packages/test-commerce-sdk-react/app/components/_app-config/index.tsx

```diff
@@ -73,12 +73,17 @@ AppConfig.restore = () => {}
 AppConfig.extraGetPropsArgs = () => {}
 AppConfig.freeze = () => {}
 
-// Recommended settings for PWA-Kit usage.
+const isServerSide = typeof window === 'undefined'
+// Recommended settings for PWA-Kit usages.
 const options = {
     queryClientConfig: {
         defaultOptions: {
             queries: {
-                retry: false
+                retry: false,
```

Comment:
I'd explicitly expose the configuration is used server-side and client-side by adding conditionals.

```
const isServer = typeof window === 'undefined'
    ...
// Recommended settings for PWA-Kit usage.
const options = {
    queryClientConfig: {
        defaultOptions: {
            queries: {
                retry: isServer? false: false
            },
            mutations: {
                retry: isServer? false: false
            }
        }
    }
}
```

Reply:
@adamraya I see the benefit of doing that. More verbose, but clearer as a result.

Reply:
Please don't do this.

Reply:
As a developer seeing this, I think I would go from feeling confident about what the configuration is, to being confused and thinking "Ummm... maybe `isServer` isn't a boolean? Why the ternary? Maybe I don't really understand this straightforward thing that I thought I understood"

Reply:
The idea of introducing friction with the ternary is to intentionally make the developer think twice before changing a setting that could work great on the client, but it could be a disaster on the server or viceversa. 

If we decide to not add conditionals, How can we make more evident to users that config options will be applied both client and server side?